examples/mnist/run.py

```diff
@@ -168,24 +164,25 @@ def receive_output(self, logits: tf.Tensor) -> tf.Operation:
   prediction_client = PredictionClient(
       player_name="prediction-client",
       local_data_file="./data/test.tfrecord")
-
+  
   # get model parameters as private tensors from model owner
-  params = tfe.define_private_input(model_owner.player_name,
-                                    model_owner.provide_input, masked=True)  # pylint: disable=E0632
-
+  params = tfe.define_private_input(model_owner.player_name, model_owner.provide_input) # pylint: disable=E0632
+  
   # we'll use the same parameters for each prediction so we cache them to
   # avoid re-training each time
   cache_updater, params = tfe.cache(params)
 
-  # get prediction input from client
-  x = tfe.define_private_input(prediction_client.player_name,
-                               prediction_client.provide_input, masked=True)  # pylint: disable=E0632
-
-  # compute prediction
-  w0, b0, w1, b1 = params
-  layer0 = tfe.matmul(x, w0) + b0
-  layer1 = tfe.sigmoid(layer0 * 0.1)  # input normalized to avoid large values
-  logits = tfe.matmul(layer1, w1) + b1
+  with tfe.protocol.SecureNN():
+    model = tfe.keras.Sequential()
+    model.add(tfe.keras.layers.Dense(512, input_shape=[PredictionClient.BATCH_SIZE,28*28]))
+    model.add(tfe.keras.layers.Activation('relu'))
+    model.add(tfe.keras.layers.Dense(10, activation=None))
+    model.set_weights(params)
```

Comment:
hmm, we might have to move `set_weights` under `with tfe.Session(target=session_target)` to make sure it's using the tfe session when assigning variables:
https://github.com/tf-encrypted/tf-encrypted/blob/9bda5c593aab39a599b5e2634a6e21e7bf01ee83/tf_encrypted/keras/engine/base_layer.py#L128-L138